src/components/homepage/Homepage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from "react-redux";
+
+export interface CardsProps {
```

Comment:
Please change to:
`export interface HomeProps {}
const HomePage = (props:HomeProps) => {`

since we don't receive any props for now